index.html

```diff
@@ -8,32 +8,171 @@
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="canopy-items">
+                    <li><img src="img\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="img\Counter.svg"></li>
+                    <li class="item">2015</li>
+                </ul>
+            </section>
+            <p class="description-work">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="programming-lan">
+                <li class="item-lan">html</li>
+                <li class="item-lan">css</li>
+                <li class="item-lan">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        <section class="section3">
+            <img class="snap" src="img\Snapshoot Portfolio 3.svg" alt="Project picture">
+            <h1 class="project-title">Tonic</h1>
+            <section class="canopy-sec">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="canopy-items">
+                    <li><img src="img\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="img\Counter.svg"></li>
+                    <li class="item">2015</li>
+                </ul>
+            </section>
+            <p class="description-work">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="programming-lan">
+                <li class="item-lan">html</li>
+                <li class="item-lan">css</li>
+                <li class="item-lan">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        <section class="section4">
+            <img class="snap" src="img\Snapshoot Portfolio 4.svg" alt="Project picture">
+            <h1 class="project-title">Multi-Post Stories</h1>
+            <section class="canopy-sec">
+                <h2 class="canopy">CANOPY</h2>
+                <ul class="canopy-items">
+                    <li><img src="img\Counter.svg"></li>
+                    <li class="item">Back End Dev</li>
+                    <li><img src="img\Counter.svg"></li>
+                    <li class="item">2015</li>
+                </ul>
+            </section>
+            <p class="description-work">
+                A daily selection of privately personalized 
+                reads; no accounts or sign-ups required.
+            </p>
+            <ul class="programming-lan">
+                <li class="item-lan">html</li>
+                <li class="item-lan">css</li>
+                <li class="item-lan">javaScript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+
+    </div>
+    <div id="about-myself">
```

Comment:
- **[OPTIONAL]** Consider changing the main container of your **About my self section** into `<section>`.